src/setup_payload/Base38Encode.h

```diff
@@ -25,10 +25,16 @@
 
 namespace chip {
 
-// out_buf is null-terminated on success
+/*
+ * The out_buf is null-terminated on success.
+ *
+ * The resulting size of the out_buf span will not include the null terminator.
```

Comment:
I think the important thing is that the resulting size of the out_buf span will be the size of the data written to the buffer, not including the null-terminator, right?